src/components/Navbar.astro

```diff
@@ -60,46 +60,85 @@ const currentPath = Astro.url.pathname;
     justify-content: center;
 
     max-width: 90%;
-    width: 35rem;
+    width: 45rem;
     z-index: 100;
     background-color: $background-secondary;
-    border-radius: 10px;
+    border-radius: 12px;
+
+    background: linear-gradient($background-secondary, $background-secondary) padding-box,
+      linear-gradient(
+          180deg,
+          rgba($border-color, 0.1) 0%,
+          rgba($border-color, 0) 83.85%,
+          rgba(white, 0.05) 100%,
+          rgba($border-color, 0.05) 100%
+        )
+        border-box;
+    border: 2px solid transparent;
+
+    box-shadow: 0px 7px 9px 2px rgba(black, 0.2);
 
     @media (max-width: #{$screen-xl-max}) {
-      width: 25rem;
+      width: 45rem;
+    }
+
+    @media screen and (orientation: landscape) and (max-device-width: $screen-md-max) {
+      width: 27rem;
     }
 
     container: Navbar / inline-size;
   }
 
   a {
     display: flex;
-    flex-direction: column;
+    flex-direction: row;
     align-items: center;
-    justify-content: flex-end;
+    justify-content: center;
 
     width: 25cqi;
     margin: min(0.6rem, 1.5cqi) min(0.8rem, 2cqi);
-    padding: min(0.8rem, 2cqi) min(1.2rem, 3cqi);
+    padding: min(0.8rem, 2cqi) min(0.6rem, 3cqi);
+    padding-left: 0;
     border-radius: 8px;
+    gap: 4px;
 
     text-decoration: none;
     transition: background-color 0.3s ease;
```

Comment:
change this to  `transition: all 0.3s ease;` so that all properties have transition. Currently it looks a bit weird because only one property has it.